internal/pkg/template/templates/workloads/partials/cf/executionrole.yml

```
@@ -41,6 +41,15 @@ ExecutionRole:
                 - 'kms:Decrypt'
               Resource:
                 - !Ref ArtifactKeyARN
+            - Effect: 'Allow'
```

nitnitnitnitnit™️ !
```suggestion
                Sid: DecryptAppRegionalArtifactKey
              - Sid: DecryptTaggedKMSKey
                Effect: 'Allow'
```
(idk if the indentation is correct, so don't "commit suggestion" just in case 😂 )